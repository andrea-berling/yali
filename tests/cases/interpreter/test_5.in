// This program tests the syntax of multiple statements in a single line
print "foo"; print true;
print false;
print "baz"; print 78;
