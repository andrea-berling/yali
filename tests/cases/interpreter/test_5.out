foo
true
false
baz
78
